docs/user/man_pages/bodhi.rst

```diff
@@ -334,6 +339,12 @@ The ``updates`` command allows users to interact with bodhi updates.
 
         The name of the update
 
+    ``--from-tag``
+
+        If given, this flag will cause that ``<update>`` will be considered as created from Koji tag
+        and will use tag returned by query instead of builds.
```

Comment:
Let's reword this: "If given, for updates that were created from a Koji tag, this will update the builds to the latest ones in the tag."